pkgs/development/python-modules/fs/default.nix

```
@@ -46,6 +47,7 @@ buildPythonPackage rec {
     mock
     psutil
     pytestCheckHook
+    glibcLocales
```

```suggestion
  ]
  ++ lib.optionals isPypy [
    glibcLocales
```
or add a comment. Otherwise someone will remove this again.

---

Done!